### PR TITLE
Add several sections from IOI2021 rules to rules.md

### PR DESCRIPTION
**Pull request by jonathanirvings**

### CODE REVIEW

File: rules.md [hunk 1]
```diff
@@ -4,74 +4,79 @@ title: APIO 2021 Competition Rules
...
 
-* A contestant can submit a clarification request in English during the competition.
-* Clarification requests will be answered only in English with one of the following:
-   * “YES”
-   * “NO”
-   * “ANSWERED IN TASK DESCRIPTION (EXPLICITLY OR IMPLICITLY)” – The task description contains sufficient information. The contestant should read it again carefully.
-   * “INVALID QUESTION” – The question is most likely not phrased so that a yes/no answer would be meaningful. The contestant is encouraged to rephrase the question.
-   * “NO COMMENT” – The contestant is asking for information that the Scientific Committee cannot give.
+* A contestant can submit a clarification request concerning competition tasks, rules, and/or grading submitted using the grading system during the competition.
+* Clarification requests may be expressed only in English and will be answered only in English.
```
Comment (prabowo02) [inline rules.md]:
```suggestion
* Clarification requests must be expressed only in English and will be answered only in English.
```

File: rules.md [hunk 2]
```diff
@@ -4,74 +4,79 @@ title: APIO 2021 Competition Rules
...
 ### D. Clarification
 
-* A contestant can submit a clarification request in English during the competition.
-* Clarification requests will be answered only in English with one of the following:
-   * “YES”
-   * “NO”
-   * “ANSWERED IN TASK DESCRIPTION (EXPLICITLY OR IMPLICITLY)” – The task description contains sufficient information. The contestant should read it again carefully.
-   * “INVALID QUESTION” – The question is most likely not phrased so that a yes/no answer would be meaningful. The contestant is encouraged to rephrase the question.
-   * “NO COMMENT” – The contestant is asking for information that the Scientific Committee cannot give.
+* A contestant can submit a clarification request concerning competition tasks, rules, and/or grading submitted using the grading system during the competition.
```
Comment (prabowo02) [inline rules.md]:
```suggestion
* A contestant can submit a clarification request concerning competition tasks, rules, and/or grading using the grading system during the competition.
```

File: rules.md [hunk 3]
```diff
@@ -4,74 +4,79 @@ title: APIO 2021 Competition Rules
...
-
-* **Creative** scoring:
-  * The scoring formula will vary for each problem and will be explicitly stated in the problem description.
+* The source program provided by the contestant must be contained in one source file as specified in the task statement.
+* Submissions must not perform explicit input and output operations; instead data must only be exchanged through the interfaces specified in the task statement.
+* For each programming task, the contestants can download a zip file for a programming task from the grading system.
+  * The zip file contains interface files, a sample grading program, and a skeleton implementation of a required source file.
+  * The skeleton exercises an interface, but it does not solve the task.
+  * The provided sample grader would not be the same as the official grader used by the grading system.
+* Each contestant may submit up to **50 submissions** for each task.
```
Comment (prabowo02) [inline rules.md]:
```suggestion
* Each contestant may submit up to **50 submissions** for each task. This limit may be increased for some task types.
```

ini perlu ga?

**Review by prabowo02 — CHANGES_REQUESTED** [review]

File: rules.md (hunk 3 above)
Comment (jonathanirvings) [reply rules.md]: Gw tambahin "unless otherwise stated in the task statement" instead.

**Review by prabowo02 — APPROVED** [review]

**Pull request merged**